packages/components/src/__actions__/Button/v3/Button.module.scss

```diff
@@ -0,0 +1,109 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "~@kaizen/design-tokens/sass/typography";
+@import "../../../../styles/utils/button-reset";
+
+$button-x-y-min: $spacing-48;
+
+// spacing-40 does not currently exist. Holding on adding this until the button rfc has been signed off
+$button-small-x-y-min: 2.5rem;
+$button-y-padding: calc(#{$spacing-8} - #{$border-solid-border-width});
+
+// $spacing-22 (1.375rem) doesn't exist in design tokens. Since this will be changes in the rebuild we shouldn't add it
+$button-x-padding: calc($spacing-12 - #{$border-solid-border-width});
+$button-x-padding-small: calc(#{$spacing-8} - #{$border-solid-border-width});
+
+.button {
```

Comment:
I'm wondering if during these rebuilds we introduce a layers to improve our specificity control 🤔 cc/ @jakubriedl

Reply:
Actually, while we're talking about potential changes to line up with rebuilds, do we want to slowly ween ourselves off SCSS and write new component as CSS? / are there any blockers to doing so. I know it was discussed a long time ago, but wondering if that's still on the cards for us?

Reply:
We need Storybook's config to separate the process of tailwind and `modules.css` or something around that space needs to fix.

Reply:
all good points, yes and yes.
Let's talk about them on next opportunity how we can plan them and get them done. But for this PR let's not baloon the scope.